pkg/antctl/output/output.go

```diff
@@ -263,6 +246,31 @@ func ConstructTable(numRows int, numCols int, widths []int, rows [][]string, wri
 	return nil
 }
 
+func ConstructFormattedTable(numRows, numCols int, rows [][]string, sortRows bool, writer io.Writer) error {
```

Comment:
Isn't `numRows` just `len(rows)`?

Reply:
Updated the function input, and comments.